startup.py

```diff
@@ -20,16 +28,89 @@ async def startup(idea: str, investment: float = 3.0, n_round: int = 5, code_rev
     await company.run(n_round=n_round)
 
 
-def main(idea: str, investment: float = 3.0, n_round: int = 5, code_review: bool = False):
+async def education_startup(lesson_file: str, investment: float = 3.0, n_round: int = 1, *args, **kwargs):
+    """Run a startup. Be a teacher in education industry."""
+
+    demo_lesson = """
+    UNIT 1 Making New Friends
+    TOPIC 1 Welcome to China!
+    Section A
+
+    1a Listen and number the following names.
+    Jane Mari Kangkang Michael
+    Look, listen and understand. Then practice the conversation.
+    Work in groups. Introduce yourself using
+    I ’m ... Then practice 1a
+    with your own hometown or the following places.
+
+    1b Listen and number the following names
+    Jane Michael Maria Kangkang
+    1c Work in groups. Introduce yourself using I ’m ... Then practice 1a with your own hometown or the following places.
+    China the USA the UK Hong Kong Beijing
+
+    2a Look, listen and understand. Then practice the conversation
+    Hello! 
+    Hello! 
+    Hello! 
+    Hello! Are you Maria? 
+    No, I’m not. I’m Jane.
+    Oh, nice to meet you, Jane
+    Nice to meet you, too.
+    Hi, Maria!
+    Hi, Kangkang!
+    Welcome to China!
+    Thanks.
+
+    2b Work in groups. Make up a conversation with your own name and the
+    following structures.
+    A: Hello! / Good morning! / Hi! I’m ... Are you ... ?
+    B: ...
+
+    3a Listen, say and trace
+    Aa Bb Cc Dd Ee Ff Gg
+
+    3b Listen and number the following letters. Then circle the letters withthe same sound as Bb.
+    Aa Bb Cc Dd Ee Ff Gg
+
+    3c Match the big letters with the small ones. Then write them on the lines.
+    """
+
+    lesson = ""
+    if lesson_file is not None and Path(lesson_file).exists():
+        async with aiofiles.open(lesson_file, mode="r", encoding="utf-8") as reader:
+            lesson = await reader.read()
+            logger.info(f"Course content: {lesson}")
+    if not lesson:
+        logger.info("No course content provided, using the demo course.")
+        lesson = demo_lesson
+
+    company = SoftwareCompany()
+    company.hire([Teacher(*args, **kwargs)])
+    company.invest(investment)
+    company.start_project(lesson, role="Teacher", cause_by=TeachingPlanRequirement)
+    await company.run(n_round=1)
+
+
+def main(idea: str, investment: float = 3.0, n_round: int = 5, *args, **kwargs):
```

Comment:
I'am not sure if it's suitable to start all `industry` only from the `startup.py`. It will make `startup.py` be more complex with lots of parameters. Maybe  @geekan can make a decision.

Reply:
canceled.